internal/certs/testdata/ca-key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
Maybe drop a CONTRIBUTING.md with instructions to generate those :)

Reply:
Added as comment in the tests.